tests/e2e/chain/chain.go

```diff
@@ -41,6 +41,9 @@ func (c *internalChain) createAndInitValidators(count int) error {
 		if err := node.createKey("val"); err != nil {
 			return err
 		}
+		if err := node.createKey("val2"); err != nil {
```

Comment:
Why do we create two keys per validator?

Reply:
I did this because I didn't want to use a validator's self bonded wallet, since if we did this, both the superfluid stake that was bonded as well as the validators vote would change. Now that the water is less murky though, I feel like I could probably do this and just calculate what the new tally result would be with the validator re-voting

Reply:
Do we need a delegator account /wallet other than the validator's? I think we should name it accordingly since having 2 validator keys per node is confusing

Reply:
Went for adding it via the test instead of during initialization, thanks for the feedback!